code/modules/mob/living/simple_animal/abnormality/aleph/white_night.dm

```diff
@@ -245,11 +246,18 @@ GLOBAL_LIST_EMPTY(apostles)
 
 /mob/living/simple_animal/hostile/apostle/death(gibbed)
 	invisibility = 30 // So that other mobs cannot attack them
+	death_counter += 1
 	return ..()
 
 /mob/living/simple_animal/hostile/apostle/revive(full_heal = FALSE, admin_revive = FALSE, excess_healing = 0)
 	invisibility = 0 // Visible again
 	can_act = TRUE // In case we died while performing special attack
+	if(death_counter == 1)
+		health = 1000
+	else if(death_counter == 2) // There has to be SOME incentive to suppressing the Apostles. Even if that's just making them easier to kill again. 2k HP with good resitsances is too much to chunk through considering they respawn with that FREQUENTLY.
+		health = 750
+	else if (death_counter >= 3)
+		health = 500
```

Comment:
```suggestion
	if(death_counter > 0) // There has to be SOME incentive to suppressing the Apostles. Even if that's just making them easier to kill again. 2k HP with good resitsances is too much to chunk through considering they respawn with that FREQUENTLY.
		health -= death_counter * 400
```

Reply:
I would personally like the first time each apostle dies to be the most "ground" they lose. Them being really scary at the start to being sligjtly manageble death machines is kinda the idea. Though I could amd probably will make this a switch statement now